frontend/utils/useFetch.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+const useFetch = (options:{}) =>{
```

Comment:
Define a custom type for your JSON data before

Reply:
I think we can use types for validating data like interfaces, but i dont think all data will look the same so we can't define type of returned json data.
Egz.
registration form will differ from user profile data.